packages/dashboard/src/lib/github.ts

```diff
@@ -1,8 +1,5 @@
 export const getGithubCommitURL = (repo: string, sha: string) =>
-  handleSshURL(repo.replace(/.git$/, '') + `/commit/${sha}`);
+  repo.replace(/.git$/, '') + `/commit/${sha}`;
 
 export const getGithubBranchURL = (repo: string, branch: string) =>
-  handleSshURL(repo.replace(/.git$/, '') + `/tree/${branch}`);
-
-export const handleSshURL = (sshUrl: string) =>
```

Comment:
hm.. why was it removed?

Reply:
What I understand and what I can see in the code this is now handled only in the director converting git URL to a HTTP/HTTPS URL. The dashboard will only get the the Branch URL as either https:// or http://

Example:
director service will convert this data 
from:
git@ts-gitlab.sorrycypress.lan:web-dev/testing_something.git 

to:
 https://ts-gitlab.sorrycypress.lan/web-dev/testing_something.git
 
 So why do this again in the dashboard?